_i18n/ar/resources/moneropedia/sidechain.md

```diff
@@ -0,0 +1,17 @@
+---
+entry: "sidechain"
+terms: ["sidechain", "side-chain", "auxillary-chain"]
+summary: "a blockchain that is merge mined with a primary blockchain"
+---
+
+{% include disclaimer.html translated="no" translationOutdated="no" %}
+
+### The Basics
+
+A sidechain (also known as an auxillary chain) is a @blockchain that is @merge-mined with Monero or other cryptocurrency. In most cases the sidechain will have a lower difficulty than the primary blockchain.
+
+A sidechain that that allows you to transfer assets between different blockchains is called a "pegged blockchain". It works by setting the value of one asset equal to the value of another asset on an alternate blockchain, otherwise known as pegging. Currently there are no pegged sidechains that exist for Monero.
```

Comment:
```suggestion
A sidechain that allows you to transfer assets between different blockchains is called a "pegged blockchain". It works by setting the value of one asset equal to the value of another asset on an alternate blockchain, otherwise known as pegging. Currently there are no pegged sidechains that exist for Monero.
```